autoware_map_msgs/README.md

```diff
@@ -29,3 +33,7 @@ Let $X_0$ be a set of PCD map ID that the client node has, $X_1$ be a set of PCD
 ( $x \in A\backslash B \iff x \in A \wedge x \notin B$ )
 
 <img src="./media/differential_area_loading.gif" alt="drawing" width="400"/>
+
+## GetSelectedPointCloudMap.srv
+
+Given IDs query that the client node already has, the response is expected to contain the PCD maps (each of which attached with unique ID).
```

Comment:
`the client node already has` part should be clarified. Maybe you should also mention that the client is expected to receive `PointCloudMapCellMetaDataWithID.msg` metadata beforehand to use this interface.

Reply:
Thanks!!
I added that information in https://github.com/autowarefoundation/autoware_msgs/pull/57/commits/565d5c76bce909a0a241c00e1a10c294c715d436